docs/react-18-and-react-native.md

```diff
@@ -0,0 +1,136 @@
+---
+id: react-18-and-react-native
+title: React 18 & React Native
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+This page describes some of the interactions between React and React Native,
+and their implications for the New Architecture of React Native.
+
+If you're interested in using React 18 in your React Native app, this page will also provide meaningful context.
+
+:::info
+**tl;dr** The first version of React Native compatible with React 18 is **0.69.0**.
+Moreover, to use the [new features of React 18 (i.e. Concurrent React)](https://reactjs.org/blog/2022/03/29/react-v18.html),
+you must migrate your React Native app to the New Architecture.
+:::
+
+## React and React Native Versions
+
+The version alignment between React and React Native relies on two syncronization points:
+
+1. The versions in the `package.json` of the new app template. For example [for React Native 0.68.1](https://github.com/facebook/react-native/blob/b5f1b26a8ff4014e45bbb95c5956ba2c2cf286db/template/package.json#L12-L15) the versions are aligned as follows:
```

Comment:
```suggestion
1. The versions in the `package.json` of the new app template. For example [for React Native 0.68.1](https://github.com/facebook/react-native/blob/0.68-stable/template/package.json#L12-L15) the versions are aligned as follows:
```

Let's use link to the the source in stable tag, instead of the latest commit.